### ISSUE TITLE
[lately][func] 계정설정, 관리자콘솔 - 사용자정보에 생일 정보 편집기능 없음. 생일정보 추가 요청

### ISSUE THREAD

**Issue by jungheekang** [vmeeting-io/vmeeting_issue#214]
[발생현상]
계정설정, 관리자콘솔 - 사용자정보에 생일 정보 편집 기능 없음. 생일정보 추가 요청

* 계정설정 
- 계정설정 추가 문구
(kor) 생일정보는 회의 내 생일 알림, 생일 모자 AR적용에만 사용됩니다.
(eng) Birthday info will only be used for birthday noti  and birthday hat AR in meeting room

<img width="425" alt="스크린샷 2021-10-15 오전 11 37 51" src="https://user-images.githubusercontent.com/87403323/137423782-299d67a1-8c08-4c2a-b92b-2e7d2b9f896b.png">

* 관라자콘솔 - 사용자정보
<img width="567" alt="스크린샷 2021-10-15 오전 11 41 24" src="https://user-images.githubusercontent.com/87403323/137424275-f3c9dff5-f477-46f8-9f99-b97fad8b928c.png">

[발생일자]
2021-10-15

[재현경로]
1. 메인사이트 - 계정 로그인 - 로그인정보 - 계정설정에 생일정보 없음

1. 메인사이트 - 관리자계정 로그인 - 관리자콘솔 - 사용자관리 - 사용자 선택 - 사용자정보에 생일 정보 없음

**Comment by sajan-dpnm** [comment]
@jungheekang the birthdate field in account settings should be editable right? but the birthdate field in admin console-> user manage should be uneditable right?

**Comment by jungheekang** [comment]
@sajan-dpnm I think both should be editable. please make it editable

**Comment by sajan-dpnm** [comment]
Code changes pushed to lately. Requesting @jungheekang for further tests

![image](https://user-images.githubusercontent.com/72065079/137687636-95a8a277-1ea3-4234-ba56-905e314b6bbe.png)

![image](https://user-images.githubusercontent.com/72065079/137687732-52eba4ac-c3c0-45a0-84b4-f75669cc5cd1.png)

**Comment by jungheekang** [comment]
@sajan-dpnm Please change the language according to the language setting

**Comment by jungheekang** [comment]
@sajan-dpnm If you delete the date by pressing the X (Delete) button in the calendar area and save it, it is displayed like an attached file image.
<img width="500" alt="스크린샷 2021-10-19 오후 12 15 11" src="https://user-images.githubusercontent.com/87403323/137838421-ccad5274-d3ed-4985-ad50-c094ef01491c.png">

**Comment by jungheekang** [comment]
@sajan-dpnm 
Meeting Room - Settings - Profile tab - birthday If you delete the date and save the data by clicking the OK button, an error occurs in Account Setting - Birthday's Calendar and Admin Console.
<img width="646" alt="스크린샷 2021-10-19 오후 12 26 33" src="https://user-images.githubusercontent.com/87403323/137839199-a40a6835-be31-4361-9a87-712813276213.png">

**Comment by sajan-dpnm** [comment]
@jungheekang that's right. Because the date is now updated to empty date, it will show as an empty string, which will result to `Invalid Date`. However, account user and Admin from the admin console can assign a new birthdate by typing in the format `YYYY-MM-DD` which will assign a new date to the participant.

**Comment by sajan-dpnm** [comment]
@jungheekang 
> @sajan-dpnm Please change the language according to the language setting

I didn't get what you meant!

**Comment by sajan-dpnm** [comment]
@jungheekang 

![screen-capture](https://user-images.githubusercontent.com/72065079/137841977-be0a2ae6-6d55-4cad-9313-fcd1f3f4f081.gif)

**Comment by jungheekang** [comment]
> @jungheekang
> 
> > @sajan-dpnm Please change the language according to the language setting
> 
> I didn't get what you meant!

If the system language is English, the calendar is displayed in English. If the system language is Korean, it is displayed in Korean.

All are now displayed in Korean.
<img width="500" alt="스크린샷 2021-10-19 오후 12 15 11" src="https://user-images.githubusercontent.com/87403323/137842450-84265535-a1c2-43de-8a63-47c5d487233f.png">
![스크린샷 2021-10-19 오후 1 10 53](https://user-images.githubusercontent.com/87403323/137842570-02a8d478-da0f-4661-8239-495778c25e60.png)

**Comment by jungheekang** [comment]
@sajan-dpnm If there is no value in the calendar input box, check it when saving to prevent it from being saved. Display an alert or previous value

**Comment by sajan-dpnm** [comment]
@jungheekang You said that birthday input is not a required field during user registration. But now you want to prevent the user from saving an empty date value?

**Comment by jungheekang** [comment]
@sajan-dpnm In a situation where the date is stored, if the user just clears the value and does not enter it,
It means to notify you that there is no date entered or to enter a value before deletion.

**Comment by jungheekang** [comment]
@sajan-dpnm 
So let's do this.
The default value for the date of birth is 1980-01-01.
If you delete the date of birth and save it, set it as the default value.
Also, please set the birthday date of all members as 1980-01-01.
Then, the birthday data of all members will be displayed on the admin console.

**Comment by sajan-dpnm** [comment]
@jungheekang okay, i will look into it
